pkg/virt-controller/services/template_test.go

```diff
@@ -630,3 +693,13 @@ func MakeFakeConfigMapWatcher(configMaps []kubev1.ConfigMap) *cache.ListWatch {
 	}
 	return cmListWatch
 }
+
+func True() *bool {
```

Comment:
These functions are probably useful in general. Is there a reasonable module we could share them from?